git-branchless-lib/src/git/repo.rs

```diff
@@ -2142,14 +2142,27 @@ impl<'repo> Branch<'repo> {
         Ok(self.inner.get().target().map(make_non_zero_oid))
     }
 
-    /// Get the name of this branch, not including any `refs/heads/` prefix.
+    /// Get the name of this branch, not including any `refs/heads/` prefix. To get the full
+    /// reference name of this branch, instead call `.into_reference().get_name()?`.
     #[instrument]
     pub fn get_name(&self) -> eyre::Result<&str> {
         self.inner
             .name()?
             .ok_or_else(|| eyre::eyre!("Could not decode branch name"))
     }
 
+    /// Get the full reference name of this branch, including the `refs/heads/` or `refs/remotes/`x
```

Comment:
Here's some super high-quality, in-depth review feedback: I think there's an an extra, unintentional "x" at the end of this line. 😜 

And 🙌 for closing 5 bugs w/ 1 PR!